policy_params:
  # Structure
  # action_name:
  #   recipe:
  #   - "skill_name1"
  #   - "skill_name2"
  #   fail_reaction:
  #   - "skill_name3"
  #   - "skill_name4"
  #   calibration:
  #   - "skill_name5"
  #   - "skill_name6"
  #   ratio: 0.8
  insert_coke:
    recipe:
    - "approach_cart_move"
    - "approach_st"
    - "first_plane_st"
    - "second_plane_st"
    - "insert_st"
    # fail_reaction:
    # - "skill_name1"
    # - "skill_name2"
    # calibration:
    # - "skill_name1"
    # - "skill_name2"
    # ratio: 0.8
